test/caper_run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+echo "Running cromwell $CROMWELL, womtool $WOMTOOL and image $TAG"
+
+if [[ $# -eq 1 ]]; then
+    caper run $1 --docker $TAG --cromwell $CROMWELL --womtool $WOMTOOL --options ./test/cromwell_options.json -m metadata.json
+fi
+
+if [[ $# -eq 2 ]]; then
+    caper run $1 --docker $TAG --cromwell $CROMWELL --womtool $WOMTOOL -i $2 --options ./test/cromwell_options.json -m metadata.json
```

Comment:
Same as above.

Reply:
But if I don't want to download these every time I run them locally?

Reply:
Caper will cache them in `~/.caper/{cromwell_jar,local_jar}`